src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -30,11 +30,11 @@ namespace mxnet {
 namespace op {
 
 // infer storage function for _identity_with_attr_like_rhs op
-static bool IdentityAttrLikeRhsStorageType(const nnvm::NodeAttrs& attrs,
-                                           const int dev_mask,
-                                           DispatchMode* dispatch_mode,
-                                           std::vector<int> *in_attrs,
-                                           std::vector<int> *out_attrs) {
+static bool InferStorage(const nnvm::NodeAttrs &attrs,
```

Comment:
I'm afraid mxnet::op::InferStorage is too generic while its implementation is only for `_identity_with_attr_like_rhs` op. This will cause confusion

Reply:
It's used only in 1 place and the problem with it is that it's debug symbol is too long which creates a warning which is treated as error. This function is not exposed outside of the translation unit as well which lowers the risk of confusion. I would be happy to rename it to anything more useful if it's short enough to silence the warning

Reply:
I changed it to `IdentLikeRhsStorageType` and moved it to anonymous namespace instead of static per rule [SF.22 in CppCoreGuideLines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rs-unnamed2)